src/components/radio/README.md

```diff
@@ -0,0 +1,52 @@
+# MdRadio
+Radio buttons allow the user to select one option from a set. Use radio buttons for exclusive selection if you think that the user needs to see all available options side-by-side.
+
+![Preview](https://material-design.storage.googleapis.com/publish/material_v_4/material_ext_publish/0Bx4BSt6jniD7Z1NaaXh2ZkpDRkE/components_switches_radio1.png)
```

Comment:
I updated a screenshot for you:
https://material.angularjs.org/material2_assets/radio/radios.png